server/service/client_policies.go

```diff
@@ -12,3 +17,42 @@ func (c *Client) CreateGlobalPolicy(name, query, description, resolution, platfo
 	var responseBody globalPolicyResponse
 	return c.authenticatedRequest(req, verb, path, &responseBody)
 }
+
+// ApplyPolicies sends the list of Policies to be applied to the
+// Fleet instance.
+func (c *Client) ApplyPolicies(specs []*fleet.PolicySpec) error {
+	req := applyPolicySpecsRequest{Specs: specs}
+	verb, path := "POST", "/api/latest/fleet/spec/policies"
+	var responseBody applyPolicySpecsResponse
+	return c.authenticatedRequest(req, verb, path, &responseBody)
+}
+
+// GetPolicies retrieves the list of Policies. Inherited policies are excluded.
+func (c *Client) GetPolicies(teamID *uint) ([]*fleet.Policy, error) {
+	verb, path := "GET", ""
+	if teamID != nil {
+		path = fmt.Sprintf("/api/latest/fleet/teams/%d/policies", *teamID)
+	} else {
+		path = "/api/latest/fleet/policies"
+	}
+	var responseBody listGlobalPoliciesResponse
```

Comment:
Nit for clarity `listTeamPoliciesResponse` depending if teamID != nil? (it works because they have the same members).

Reply:
Yes, same as above.